backends/ebpf/target.h

```diff
@@ -53,6 +59,34 @@ class Target {
     virtual void emitTableDecl(Util::SourceCodeBuilder* builder,
                                cstring tblName, TableKind tableKind,
                                cstring keyType, cstring valueType, unsigned size) const = 0;
+    virtual void emitTableDeclSpinlock(Util::SourceCodeBuilder* builder,
+                               cstring tblName, TableKind tableKind,
+                               cstring keyType, cstring valueType, unsigned size) const {
+        (void) builder;
```

Comment:
how about actually giving a warning that on this architecture the table is not supported?
I suspect that this is what it amounts to.